src/api/server/authn.rst

```diff
@@ -282,7 +282,7 @@ Proxy Authentication
         [chttpd]
         authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, proxy_authentication_handler}, {chttpd_auth, default_authentication_handler}
 
-`Proxy authentication` is very useful in case your application already uses
+``Proxy authentication`` is very useful in case your application already uses
```

Comment:
I am wondering ``...`` quote work better for literal code, config or API parameters. For section names or concepts italicization might read better?

```
`Proxy authentication` 
```
<img width="491" alt="Screen Shot 2021-07-07 at 2 00 14 PM" src="https://user-images.githubusercontent.com/211822/124807414-bc429500-df2b-11eb-90ab-da4689da603b.png">

vs
```
``Proxy authentication``
```
<img width="541" alt="Screen Shot 2021-07-07 at 2 00 22 PM" src="https://user-images.githubusercontent.com/211822/124807424-bea4ef00-df2b-11eb-85e7-a11d08e45abd.png">

Since ``Proxy authentication`` is not a config parameter.

The same thing applies to `Basic Auth`.

Reply:
This was intentionally italics. Not 100% of all instances of a single backtick should be replaced with double back-ticks.